src/player.cpp

```diff
@@ -12896,6 +12894,10 @@ bool player::wearing_something_on(body_part bp) const
     return false;
 }
 
+bool player::is_wearing_item (const item& it) const {
+   return std::any_of(worn.begin(), worn.end(), [&](const item& elem) { return elem.invlet == it.invlet; });
```

Comment:
Comparison by `invlet` sounds really hacky to me.
It would probably be better to compare pointers here.